pages/roadmap.md

```diff
@@ -1,72 +1,82 @@
+- There are lots of little issues that need resolving from the launch of v2. Anything tagged with `type/bug` on the exercism/exercism repo needs sorting. You can explore that list [here](https://github.com/exercism/exercism/issues?q=is%3Aissue+is%3Aopen+label%3Atype%2Fbug).
+- **Fix issues with exercises not being unlocked:** [#3867](https://github.com/exercism/exercism/issues/3867).
 
-- **Add a button for mentors to report learners** [#4306](https://github.com/exercism/exercism/issues/4306).
+### Teams
 
-- **Fix issues with exercises not being unlocked:** [#3867](https://github.com/exercism/exercism/issues/3867).
+We are building a dedicated site for teams. It will allow groups of people to complete exercises and mentor each other in a private space. We envision it being used by bootcamps, groups of friends, and colleagues at work. Teams on Exercism v1 was a really important feature for many people so restoring it is a top priority for us. We are close to launching and ironing out the last bugs.
 
-### One week - two months
+### Abuse and quality control
 
-- **Mentoring UI Improvements:**
-  - Exposing the communal mentoring notes from GitHub to be visible directly within the UI.
-  - Adding personalised mentor notes per exercise, viewable and editable within the UI. [#4083](https://github.com/exercism/exercism/issues/4083)
-  - Adding 'Comment and Request changes' to mentoring UI, which will prompt the learner to submit another iteration (a little like Code Review on GitHub)
-  - Adding difficulty filter to solution selector [#4105](https://github.com/exercism/exercism/issues/4105)
-  - Showing more information (such as track backlog) on the dashboard [#4032](https://github.com/exercism/exercism/issues/4032)
-  - Improve mentoring notifications [#3883](https://github.com/exercism/exercism/issues/3883)
-  - Allow mentors to view other mentors' comments [#3988](https://github.com/exercism/exercism/issues/3988)
-  - Allow track mentors to find previous reviews of other solutions of the same exercise [#4364](https://github.com/exercism/exercism/issues/4364)
-  - Calculate mentors' "weekly progress" per track [#4438](https://github.com/exercism/exercism/issues/4438)
+- **Add a button for mentors to report learners** [#4306](https://github.com/exercism/exercism/issues/4306).
 
-- **Make the steps of the installation guide linkable:** [#4470](https://github.com/exercism/exercism/issues/4470)
+### Curriculum
 
-- **Add more information about timescales for mentoring:** [#4368](https://github.com/exercism/exercism/issues/4368).
+We are actively working to explore what an examplar syllabus would look like for the Ruby track, considering things such as  learning pathways, goals, mentoring-burden. We will then explore where the gaps are and what new exercises might be created as part of this. Once we have made some basic progress we will widen the discussion to include a small set of sample tracks with active maintainers.
```

Comment:
`example` would be a less archaic form of `examplar`.

Reply:
I think they mean different things. You normally have one exemplar and multiple examples. The exemplar is like the reference example.

Reply:
`examplar` doesn't appear as a word on dictionary.com or thesaurus.com.  Is `exemplar` what you are going for?
What I found for exemplar at [dictionary.com](https://www.dictionary.com/browse/exemplar) and at [thesaurus.com](https://www.thesaurus.com/browse/exemplar).

You have to pardon my American English.  Before today, I had never seen `exemplar` before.  I had to look it up.